appf.adoc

```diff
@@ -467,7 +467,7 @@ to a reference direction.
     Corresponds to an OGC WKT VERT_DATUM name. This attribute and **`geoid_name`** cannot 
     both be specified.
 
-| **`grid_mapping_name`** | N
+| **`grid_mapping_name`** | S
 | The name used to identify the grid mapping.
```

Comment:
Is this PR a good place to replace "grid mapping" with "coordinate reference system"?

Reply:
@ajelenak-thg I think that's probably part of a bigger change than this one location, isn't it? I expect we should address that separately.

Reply:
I agree.